.github/workflows/ci.yml

```diff
@@ -16,32 +16,14 @@ concurrency:
 jobs:
 
 #----------------------------------------------------------------------------
-# Default versions: Set default version for all dependencies
+# Retrieve JSON config file
 #----------------------------------------------------------------------------
-  default_versions:
+  config_file:
```

Comment:
I dont like this job name. config file are aleady a thing in f3d, lets avoid confusion.